src/Symfony/Bridge/Twig/Tests/Extension/TranslationExtensionTest.php

```diff
@@ -71,6 +71,24 @@ public function testTransChoiceComplexBody()
         $output = $this->getTemplate("{% transchoice count %}\n{{ 1 + 2 }}{% endtranschoice %}")->render();
     }
 
+    /**
+     * @expectedException        \Twig_Error_Syntax
+     * @expectedExceptionMessage Positional arguments cannot be used after named arguments for filter "trans" in "index".
+     */
+    public function testTransPositionalArgumentsAfterNamedArguments()
+    {
+        $output = $this->getTemplate('{{- "foo"|trans(arguments = {}, "custom") }}')->render();
```

Comment:
this test has nothing to do here. You are not testing the TranslationExtension here. You are testing Twig itself (where it is already tested).